Formula/sratoolkit.rb

```diff
@@ -37,93 +28,56 @@ class Sratoolkit < Formula
-                            *with_formula_args
-      ENV.deparallelize { system "make" }
-    end
+    # Need to use HDF 1.10 API: error: too few arguments to function call, expected 5, have 4
+    # herr_t h5e = H5Oget_info_by_name( self->hdf5_handle, buffer, &obj_info, H5P_DEFAULT );
+    ENV.append_to_cflags "-DH5_USE_110_API"
 
-    # Fix the error: ld: library not found for -lmagic-static
-    # Upstream PR: https://github.com/ncbi/sra-tools/pull/105
-    inreplace "tools/copycat/Makefile", "-smagic-static", "-smagic"
+    # Workaround to allow clang/aarch64 build to use the gcc/arm64 directory
+    ln_s "../gcc/arm64", buildpath/"ncbi-vdb-source/interfaces/cc/clang/aarch64" if Hardware::CPU.arm?
 
-    # Fix detection of hdf5 library on macOS as Apple Clang linker doesn't
-    # allow mixing static (-Wl,-Bstatic) and dynamic (-Wl,-Bdynamic) libraries
-    inreplace "setup/konfigure.perl", "-Wl,-Bstatic -lhdf5 -Wl,-Bdynamic", "-lhdf5" if OS.mac?
+    # Disable building tests
+    inreplace "CMakeLists.txt", "add_subdirectory( test )", ""
 
-    # Fix the error: utf8proc.o: linker input file unused because linking not done
-    # Upstream issue: https://github.com/ncbi/sra-tools/issues/283
-    if OS.linux?
-      inreplace "tools/driver-tool/utf8proc/Makefile",
-                "$(CC) $(LDFLAGS) -shared",
-                "#{ENV.cc} $(LDFLAGS) -shared"
+    # Remove SSE4.2 optimization if needed
+    if !Hardware::CPU.intel? || !MacOS.version.requires_sse42?
+      bitmagic_opt = Hardware::CPU.arm? ? "-DDBMNEONOPT" : "-DBMSSE2OPT"
+      inreplace "tools/sharq/CMakeLists.txt", "add_definitions(-msse4.2 -DBMSSE42OPT)",
+                                              "add_definitions(#{bitmagic_opt})"
     end
 
-    system "./configure", "--prefix=#{prefix}",
-                          "--build=#{buildpath}/sra-tools-build",
-                          "--with-ngs-sdk-prefix=#{ngs_sdk_prefix}",
-                          "--with-ncbi-vdb-sources=#{ncbi_vdb_source}",
-                          "--with-ncbi-vdb-build=#{ncbi_vdb_build}",
-                          *with_formula_args
-    system "make", "install"
+    system "cmake", "-S", "ncbi-vdb-source", "-B", "ncbi-vdb-build",
+                    "-DNGS_INCDIR=#{buildpath}/ngs/ngs-sdk",
+                    *std_cmake_args
+    system "cmake", "--build", "ncbi-vdb-build"
+
+    system "cmake", "-S", ".", "-B", "sra-tools-build",
+                    "-DVDB_BINDIR=#{buildpath}/ncbi-vdb-build",
+                    "-DVDB_LIBDIR=#{buildpath}/ncbi-vdb-build/lib",
+                    "-DVDB_INCDIR=#{buildpath}/ncbi-vdb-source/interfaces",
+                    *std_cmake_args
+    system "cmake", "--build", "sra-tools-build"
+    system "cmake", "--install", "sra-tools-build"
+
+    # lib64 is hardcoded for CMake install. Move files to lib and add symlink.
+    prefix.install prefix/"lib64" => "lib"
+    prefix.install_symlink "lib" => "lib64"
```

Comment:
Open to alternatives here for dealing with `lib64`.

Reply:
can we patch it in the CMake file?

Reply:
I patched it locally. For now, just created upstream issues. Previous PR attempts didn't get any responses upstream.